apps/src/p5lab/spritelab/CoreLibrary.js

```diff
@@ -868,4 +875,42 @@ export default class CoreLibrary {
   runBehaviors() {
     this.behaviors.forEach(behavior => behavior.func({id: behavior.sprite.id}));
   }
+
+  drawHeadings() {
+    let yCursor = OUTER_MARGIN;
+    if (this.heading) {
+      const x = PLAYSPACE_SIZE / 2;
+      const y = yCursor;
+      const size = this.getScaledFontSize(this.heading, 50);
+      this.drawLine(this.heading, size, x, y);
+      yCursor += LINE_HEIGHT / 2;
+    }
+    if (this.subHeading) {
+      const x = PLAYSPACE_SIZE / 2;
+      const y = yCursor;
+      const size = this.getScaledFontSize(this.subHeading, 16);
+      this.drawLine(this.subHeading, size, x, y);
+    }
+  }
+
+  drawLine(text, size, x, y) {
+    this.p5.textAlign(this.p5.CENTER);
+    this.p5.fill('black');
+    this.p5.textSize(size);
+    this.p5.text(text, x, y);
+  }
+
+  getScaledFontSize(text, desiredSize) {
```

Comment:
Maybe a note that this dupes PoetryLibrary logic in case we want to refactor at some point?